model/fn-execution/src/main/proto/beam_provision_api.proto

```diff
@@ -81,4 +81,8 @@ message ProvisionInfo {
     // (optional) A set of capabilities that this SDK is allowed to use in its
     // interactions with this runner.
     repeated string runner_capabilities = 12;
+
+    // (optional) Runtime environment metadata that are static throughout the
+    // pipeline execution.
+    map<string, string> metadata = 13;
```

Comment:
This seems redundant with pipeline options; couldn't we just use that?

Reply:
The info I'm trying to pass such as job_id and job_name are results from querying dataflow metadata server and I felt a bit weird to embed them into pipeline_options which was passed from SDK and we don't parse and alter it in provision process.